Documentation/sources/exd-cbe-beta/testapp-setup.md

```diff
@@ -0,0 +1,23 @@
+# Test app setup
+
+## Setup
+
+Please follow the below setup steps for iOS apps:
+
+1. Install Xcode - https://developer.apple.com/xcode/
```

Comment:
seems like cocoapods is also a prereq for this particular set of instructions.  are we good with that?

Reply:
Yeah, this has always been tricky. I'll just checkin the pods here.